### PR TITLE
SAK-50346 calendarwc Show today's events by default

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-50346

### CODE REVIEW

File: webcomponents/bundle/src/main/bundle/calendar-wc.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
 widget_title=Calendar
-pinned_sites_message=Showing all events for your pinned sites
+pinned_sites_message=Showing events from one year ago to a year from now, for your pinned sites
```
Comment (ottenhoff) [inline webcomponents/bundle/src/main/bundle/calendar-wc.properties:2]:
```suggestion
pinned_sites_message=Showing events for the next year, for pinned sites only
```

Comment (adrianfish) [reply webcomponents/bundle/src/main/bundle/calendar-wc.properties:2]: Are you not showing events for a 2 year window?

// Create a time range from one year ago to one year from now
        Instant oneYearAgo = now.minus(365, ChronoUnit.DAYS);
        Instant oneYearLater = now.plus(365, ChronoUnit.DAYS);

        return m_timeService.newTimeRange(oneYearAgo, oneYearLater);

Comment (ottenhoff) [reply webcomponents/bundle/src/main/bundle/calendar-wc.properties:2]: yes but most people just care about their calendar going forward not backward

Comment (adrianfish) [reply webcomponents/bundle/src/main/bundle/calendar-wc.properties:2]: Fair enough.

**Pull request merged**